src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,272 @@
+    <string name="preference_title_upload_location">上传位置</string>
+    <string name="preference_title_upload_data">允许蜂窝数据</string>
+
+    <string name="preference_summary_server">选择服务器</string>
+    <string name="preference_summary_share">选择分享</string>
+    <string name="preference_summary_location">选择地点</string>
+
+    <string name="preference_message_no_server">"未选择服务器"</string>
+    <string name="preference_message_no_share">"未选择共享"</string>
+
+    <string name="preference_entry_server_connection_auto">自动侦测</string>
+    <string name="preference_entry_server_connection_remote">远程</string>
+    <string name="preference_entry_server_connection_local">局域网</string>
+
+    <string name="preference_key_about_feedback" translatable="false">应用反馈</string>
+    <string name="preference_key_about_rating" translatable="false">应用等级</string>
+    <string name="preference_key_about_version" translatable="false">应用版本</string>
+    <string name="preference_key_about_intro" translatable="false">应用介绍</string>
+    <string name="preference_key_account_sign_out" translatable="false">帐户登出</string>
+    <string name="preference_key_server_connection" translatable="false">服务器连接</string>
+    <string name="preference_key_server_connection_auto" translatable="false">汽车</string>
+    <string name="preference_key_server_connection_remote" translatable="false">远程</string>
+    <string name="preference_key_server_connection_local" translatable="false">本地</string>
+    <string name="preference_key_tell_a_friend" translatable="false">告诉一个朋友</string>
+    <string name="preference_screen_key_upload" translatable="false">自动上传画面</string>
+    <string name="preference_key_upload_switch" translatable="false">自动上传开关</string>
+    <string name="preference_key_upload_server" translatable="false">自动上传服务器</string>
+    <string name="preference_key_upload_share" translatable="false">自动上传分享</string>
+    <string name="preference_key_upload_location" translatable="false">自动上传位置</string>
+    <string name="preference_key_upload_data" translatable="false">自动上传数据</string>
+    <string name="preference_key_language" translatable="false">语言</string>
+    <string name="pref_title_server_select">选择服务器</string>
+    <string name="pref_title_server_select_desc">选择要访问其共享的服务器。</string>
+    <string name="pref_title_server_active_list">活动服务器列表</string>
+    <string name="pref_server_active_list_desc">选择要访问其共享的活动服务器。</string>
+    <string name="pref_server_select_key" translatable="false">选择服务器</string>
+
+    <!-- Strings related to SignOut Preference-->
+    <string name="pref_title_sign_out">登出</string>
+    <string name="pref_title_account">帐户</string>
+    <string name="pref_sign_out_desc">从Amahi TV登出</string>
+    <string name="pref_action_not_defined">动作未定义</string>
+
+    <!--Strings related to Theme Preference -->
+    <string name="pref_title_light_theme">轻主题</string>
+    <string name="pref_key_light_theme" translatable="false">启用浅色主题</string>
```

Comment:
@Varinder-KM Please remove the translations for the strings marked `translatable:false`

Reply:
OK,I do